herring/puzzles/discordbot.py

```diff
@@ -230,6 +230,30 @@ async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
             target_channel = message.channel_mentions[0]
             await self.add_user_to_puzzle(payload.member, target_channel.name)
 
+        # TODO: Merge this with the above when not in ultra-conservative mode
```

Comment:
Go ahead and merge it, and eliminate the requirement that it be the bot's message while you're at it? (Per separate discussion about this being a desirable feature.) So that just any SIGNUP_EMOJI react, to any message the bot can see that mentions a puzzle channel, counts.

Reply:
Oh sorry, this is what Paul already fixed in #91. Rebase on top of that instead, presumably.